packages/backend/src/open_payments/quote/service.ts

```diff
@@ -159,6 +163,14 @@ async function createQuote(
     if (isQuoteError(err)) {
       return err
     }
+
+    if (
+      err instanceof PaymentMethodHandlerError &&
+      err.code === PaymentMethodHandlerErrorCode.QuoteNonPositiveReceiveAmount
+    ) {
+      return QuoteError.NonPositiveReceiveAmount
+    }
```

Comment:
Handle gracefully, avoid returning 500 for an unhandled `PaymentMethodHandlerError`.
This is also prepping for having better error codes in the RS in general.